src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue

```diff
@@ -257,7 +258,8 @@ const menuItemsToRender = computed<MenuItemAttrsProps[]>(() => itemsAsArray.valu
       ...(item.suffixAttrs || {}),
       icon,
     },
-    ...(item.tag ? {} : {
+    ...menuItemAttrs,
+    ...(menuItemAttrs?.onClick || menuItemAttrs?.href || menuItemAttrs?.tag ? {} : {
```

Comment:
Initially, I was confused about this syntax...
WDYT about wrapping this `logical` statement into parenthesis? 

```suggestion
    ...((menuItemAttrs?.onClick || menuItemAttrs?.href || menuItemAttrs?.tag) ? {} : {
```
Or maybe write it like 
```suggestion
    ...(['onClick', 'href', 'tag'].some((key) => key in menuItemAttrs) ? {} : {
```

Reply:
:thank_you: solved